### ISSUE TITLE
Campaigns Editor panel should be labeled as "Campaign" and not "Document"

### ISSUE THREAD

**Issue by kjohnson** [impress-org/givewp#5295]
## Expected Behavior

The main sidebar panel in the Block Editor should use the Custom Post Type `singular_name` label, with "Document" as a fallback. The `singular_name` is set, but not showing in the Block Editor.

```
const documentLabel = useSelect( ( select ) => {
	const currentPostType = select( 'core/editor' ).getCurrentPostType();
	const postType = select( 'core' ).getPostType( currentPostType );

	return (
		postType?.labels?.singular_name ??
		__( 'Document' )
	);
} );
```
https://github.com/WordPress/gutenberg/blob/8bfa69b424ac0cfbaa8290d7967bfacd5e653dd8/packages/edit-post/src/components/sidebar/settings-header/index.js#L14

## Visuals

| Post | Page | Custom Post Type |
| --- | --- | --- |
| ![image](https://user-images.githubusercontent.com/10858303/94058961-c2f28800-fdaf-11ea-8d1b-8e89d348a30c.png) | ![image](https://user-images.githubusercontent.com/10858303/94058961-c2f28800-fdaf-11ea-8d1b-8e89d348a30c.png) | ![image](https://user-images.githubusercontent.com/10858303/94058961-c2f28800-fdaf-11ea-8d1b-8e89d348a30c.png) |

## Additional Context

- https://github.com/WordPress/gutenberg/pull/17311

## Acceptance Criteria

-   [ ] Campaigns CPT sidebar label is "Campaign"

**Comment by kjohnson** [comment]
This seems to be a bug in Gutenberg.

**Comment by kjohnson** [comment]
I pulled down the Gutenberg repo. The build works as expected, so maybe this is an unreleased feature.

**Comment by kjohnson** [comment]
Looks like the `edit-post` module isn't the latest version. WP 5.5-beta-4 shipped with edit-post v3.21.3 and I think the document label feature may have just missed that cut off.

See https://github.com/WordPress/WordPress/commit/8e433f2a57578159f7d32dd781c50e82ff86797d
